src/main/java/mate/academy/dto/user/UserLoginRequestDto.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.dto.user;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.Size;
+
+public record UserLoginRequestDto(
+        @NotEmpty
+        @Size(min = 8, max = 50)
+        @Email
+        String email,
+        @NotEmpty
+        @Size(min = 8, max = 32)
```

Comment:
The same with the password
```suggestion
        @Size(min = 6, max = 32)
```